signer/src/storage/model.rs

```diff
@@ -181,25 +180,23 @@ pub struct StacksTransaction {
 #[derive(Debug, Clone, Hash, PartialEq, Eq, PartialOrd, Ord)]
 pub struct TransactionIds {
     /// Transaction IDs.
-    pub tx_ids: Vec<Vec<u8>>,
+    pub tx_ids: Vec<[u8; 32]>,
```

Comment:
Yeah, the issue here is with the struct itself, which can contain either stacks or bitcoin transaction IDs. The solution would be to create a generic txid alias, a generic TxId type, or a new struct so that we could specialize between the two. I don't like the first two options and the third felt like too much unnecessary work (which this PR kind-of qualifies for already 😅).

Reply:
I guess the overkill alternative is to create a wrapper enum that is either a bitcoin txid or a stacks txid, and can dereference to `[u8; 32]`. But that can definitely be left as a future enhancement.

Reply:
Oh right, I didn't even think of that.